tracer/src/Datadog.Trace/Propagators/DistributedContextExtractor.cs

```diff
@@ -59,7 +59,8 @@ private DistributedContextExtractor()
                 traceId = GetFullTraceId((ulong)traceIdLower, traceTags);
             }
 
-            spanContext = new SpanContext(traceId, parentId, samplingPriority, serviceName: null, origin, rawTraceId, rawSpanId)
+            // we don't consider contexts coming from this as "remote" as it could be from a version conflict scenario crossing an AppDomain boundary
```

Comment:
Version conflict happens when multiple versions of `Datadog.Trace.dll` are loaded into the _same_ `AppDomain`, so it's not crossing _that_ boundary. (Sorry for the confusion.)

```suggestion
            // we don't consider contexts coming from this as "remote" as it could be from a version conflict scenario
```

Reply:
It looks like this change didn't make it in? Trying again...

```suggestion
            // we don't consider contexts coming from this as "remote" as it could be from a version conflict scenario
```

Reply:
Thanks for catching it twice :)

I don't know what happened, maybe a messed up with what branch I was on when committing it or undid it during one of the merge conflicts. 

I'll click on the commit suggestion here to make sure